src/uwtools/config/tools.py

```diff
@@ -38,6 +37,57 @@ def compare_configs(
     return cfg_a.compare_config(cfg_b.data)
 
 
+def config_check_depths_dump(config_obj: dict, target_format: str) -> None:
+    """
+    Check that the depth of the input config does not exceed the target format's max.
+
+    :param config_obj: The reference config object.
+    :param target_format: The target format.
+    :raises: UWConfigError on excessive input-config depth.
```

Comment:
Here and below, `input-config depth` -> `reference-config depth`, given that you refer to `config_obj` above as `The reference config object`? Or just `config depth` or even just `depth`, given that the function name, `config_check_depths_dump` provides the context that we are checking depths on configs?

Reply:
I'm guessing that by `input config` in the docstring's first line you mean "input" as in "input to this function"? I think this may be confusing, as we use terms like "input" and "output" elsewhere to refer to inputs and outputs of bigger operations, like rendering a template or realizing or validating a config. Could we maybe just call it "config" or "config object" to match the terminology of the argument in question, `config_obj`? Or people sometimes say things like "..of the given config object..." where "given" (or "provided" or "specified" etc.) implies that we're talking about an argument value.

Reply:
Mmm, making it just `depth` clears a lot of that up on its own. Yes, that arg text should be updated - actually, the log message should as well since it also says 'input'